tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -703,52 +703,119 @@ def release_batch(result: ScheduledRequests | None):
-                    f"[Autotuner] Cache size after warmup is {len(AutoTuner.get().profiling_cache)}"
-                )
+            if self._torch_compile_piecewise_cuda_graph and self._torch_compile_enabled:
+                piecewise_cuda_graph_num_tokens = sorted(
+                    self._piecewise_cuda_graph_num_tokens, reverse=True)
 
-            AutoTuner.get().print_profiling_cache()
+                with capture_piecewise_cuda_graph(True):
+                    for num_tokens in piecewise_cuda_graph_num_tokens:
+                        with self.no_cuda_graph():
+                            with release_batch(
+                                    get_torch_compile_warmup_request(
```

Ruff (0.12.2)

Undefined name `get_torch_compile_warmup_request`

(F821)